### DISCUSSION TITLE
Filter on missing property

### DISCUSSION THREAD

**Discussion by svenvanoverbeke** [datalust/seq-tickets#1149] [Help] [answered]
Hello,

Can someone help me create a filter express where a specific property is missing?

So something like:

Application = 'MyApplication' and @Properties["CustomProperty"] = null

Thank you in advance
kr
Sven

**Comment by nskerl** [comment, marked as answer]
I think `not Has(property)` is what you want.

**Reply by svenvanoverbeke** [reply]
Yes, thank you very much!
